Source/WebKit/UIProcess/Cocoa/WebViewImpl.mm

```diff
@@ -132,6 +132,7 @@
 #import <pal/spi/mac/NSTextInputContextSPI.h>
 #import <pal/spi/mac/NSViewSPI.h>
 #import <pal/spi/mac/NSWindowSPI.h>
+#import <pal/spi/mac/VisionKitCoreSPI.h>
```

Comment:
This looks like it would break iOS build. The other `pal/spi/mac` headers included here have `#if PLATFORM(MAC)` or `#if USE(APPKIT)`.

Reply:
This whole file is Mac-only, it's for the `WebView` NSView subclass.

Reply:
I see. What a brilliant location for a Mac only file.